src/database.ts

```diff
@@ -2951,6 +2962,9 @@ class Database extends common.GrpcServiceObject {
             if (session) {
               session.lastError = err as grpc.ServiceError;
             }
+            span.addEvent('No session available', {
+              'failed_session.id': session?.id,
```

Comment:
keep the name same "session.id" instead of "failed_session.id"

Reply:
So "session.id" didn't make sense given the event "No session available", "failed_session.id" gave more prescription that it failed when we tried to retrieve using that sessionID, but no biggie I am making that change.